lib/src/module.ts

```diff
@@ -45,7 +47,7 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Add plugin
     nuxt.options.runtimeConfig.public.storyblok = options;
-    if (options.usePlugin !== false) {
+    if (!(options.usePlugin === false || options.enableSudoMode === true)) {
```

Comment:
The given statement is a little bit difficult to read, I suggest changing it to something simpler like:

```ts
if (options.usePlugin === true && options.enableSudoMode === false)
```

Reply:
The condition you suggest is not doing the same, the idea is to only have true if both conditions are false, otherwise if any of them are true or both should be false.

I can change it by `!(usePlugin === false) && !(enableSudoMode === true)` if you think that clarifies things for you 🫂

Reply:
Same as `!(usePlugin === false) && !(enableSudoMode === true)` or `usePlugin !== false && enableSudoMode !== true`.

Reply:
True, I just realized what was the confusion from my side, I didn't see the conditions were changed XD I was seeing this the whole time: `options.usePlugin === false && options.enableSudoMode === true` 🤣